src/components/Branding/Logo/Logo.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTheme } from '@mui/material/styles';
+
+import { LogoLarge, LogoMaximized, LogoSmall } from '../raw-icons';
+
+interface Props {
+  variant?: string;
+}
+
+const Logo: React.FC<Props> = (svgProps) => {
+  const theme = useTheme();
+  const color = theme.palette.logo;
+
+  switch (svgProps.variant) {
+    case 'maximized':
+      return <LogoMaximized sx={{ color: color, height: 53, width: 154 }} />;
```

Comment:
I'm still passing in the height and the width since the original dimensions from the svg is not being kept.  It defaults to approx. `21 x 21` instead when no height or width was passed in.